salt/defaults.yaml

```diff
@@ -29,11 +29,16 @@ salt:
   salt_api: salt-api
   salt_ssh: salt-ssh
 
+  pkgrepo: None          # see osfamilymap
+  ssh_roster: None       # see pillar data
```

Comment:
These will both evaluate to the string `None`.  If you mean Python `None`, then you'll need to set it to YAML `null` or `~`, as discussed here: https://github.com/saltstack-formulas/postgres-formula/pull/289#discussion_r323172288.

Reply:
thanks for that.